### PR TITLE
Changed uint80 to uint

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix: #39 
Changed uint80 to uint.
Linter fixes.
ERC165 identifier updated.
Regenerated docs.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<!--redirect to /README-->
```
Comment (dev-matan-tsuberi) [inline docs/index.html]: Why was this removed?

File: docs/README.md [hunk 2]
```diff
@@ -1,3 +0,0 @@
-# Welcome to DAOstack Access Control Docs
```
Comment (dev-matan-tsuberi) [inline docs/README.md]: Why was this removed?

Comment (ben-kaufman) [reply docs/README.md]: I don't know. I just re-generated the docs...

Comment (dev-matan-tsuberi) [reply docs/README.md]: `yarn docs:<..>` only touches `docs/generated_docs`

Comment (ben-kaufman) [reply docs/README.md]: I might have accidentally deleted them... I added them back.
